src/olympia/discovery/tests/test_admin.py

```diff
@@ -301,7 +301,7 @@ def test_can_add_primary_hero_with_discovery_edit_permission(self):
         assert item.custom_addon_name == 'Xäxâxàxaxaxa !'
         assert item.recommendable is True
         hero = PrimaryHero.objects.last()
-        assert hero.image == 'Ubo@2x.jpg'
+        assert hero.select_image == 'Ubo@2x.jpg'
         assert hero.gradient_color == '#054096'
         assert hero.disco_addon == item
```

Comment:
We also need some tests that prove you can create, edit, and delete the `PrimaryHeroImage`s in django admin.  Basically, look at one of the existing TestXXX classes (e.g. `TestSecondaryHeroShelfAdmin`) and create a copy of it that tests `PrimaryHeroImage` instead.